ext/ffi/tests/008.phpt

```diff
@@ -34,5 +34,5 @@ int(3)
 0 => 0
 1 => 10
 2 => 20
-FFI\Exception: Attempt to count() on non C array
+TypeError: Parameter must be an array or an object that implements Countable
```

Comment:
Hm, why does this change? That seems odd.

Reply:
Should be fixed by https://github.com/php/php-src/commit/d266ba4f2de013234eb70737f77d42941c0fec1e.